src/features/request/container/request.tsx

```diff
@@ -0,0 +1,164 @@
+import React, { FC, useCallback, useEffect, useState } from 'react';
+import { Button, Flex, Paper, Text } from '@mantine/core';
+import { isEmpty } from 'lodash';
+
+import httpService from 'shared/services/http.service';
+import { API_CONFIG } from 'shared/constants/api';
+import { IProjectsProps } from 'features/project/interface/project';
+
+import {
+    IChangeTimeSheet,
+    ILeaveRequestProps,
+    IUpComingLeave,
+} from '../interface/request';
+import LeaveRequest from '../components/leaveRequest';
+import ChangeTimeSheet from '../components/changeTimeSheet';
+
+const Request: FC<IProjectsProps> = ({ uId }) => {
+    const [leaveRequest, setLeaveRequest] = useState({} as ILeaveRequestProps);
+    const [changeTimeSheet, setChangeTimeSheet] = useState(
+        {} as IChangeTimeSheet
+    );
+    const [isVacational, setIsVacational] = useState(false);
+    const [isDisableDate, setIsDisableDate] = useState([]);
+    const [isUpcomingLeave, setIsUpcomingLeave] = useState<IUpComingLeave[]>();
+
+    const REQUEST_ARR = [
+        {
+            name: 'Request A Leave',
+            title1: ' Please submit your leave request for approval.',
+            onClick: () => {
+                setLeaveRequest({
+                    startDay: null,
+                    endDay: null,
+                    duration: '',
+                    reason: '',
+                    leaveType: '',
+                });
+            },
+        },
+        {
+            name: 'Add Missing Day',
+            title1: 'Please add the missing day to your schedule. Thank you!',
+        },
+        {
+            name: 'Change Time Request',
+            title1: 'Please submit your change time request for us to process. Thank you!',
+            onClick: () => {
+                setChangeTimeSheet({
+                    date: null,
+                    changeTimeType: '',
+                    time: '',
+                });
+            },
+        },
+        {
+            name: 'Work From Home',
+            title1: 'Enjoy the flexibility of working from home and boost your productivity.Stay connected and thrive',
+        },
+    ];
```

Comment:
Add REQUEST_ARR in constants folder

Reply:
It is not possible due to the onclick event in  REQUEST_ARR.